files/usr/lib/jeos-firstboot

```diff
@@ -352,16 +351,13 @@ for p in /sys/class/net/* ; do
         eval `wicked test dhcp4 "$d" 2>/dev/null | grep -E "^IPADDR=|^GATEWAYS="`
         ip link set down "$d" # set link down after probe once done
 
-        test -n "$IPADDR"   && candidates+=("$d")  || continue # ok, track it
-        test -n "$GATEWAYS" && candidates=("$d")  || continue # just use it
-        break
-done
-
-for d in ${candidates[@]} ; do
-        rm -f "/etc/sysconfig/network/ifcfg-$d" || exit 1
-        printf "STARTMODE=auto\nBOOTPROTO=dhcp\n" \
-                > "/etc/sysconfig/network/ifcfg-$d"
-        break # one is enough
+	# Create a configuration file for each interface that provides
+	# IPADDR and GATEWAYS
+	if [ -n "$IPADDR" -a -n "$GATEWAYS" ]; then
```

Comment:
Remove the check for `GATEWAYS` here, interfaces without routes might still be useful.

Reply:
Done. Note that for cleaning I also removed the `unset` and the `eval` parts.